app/Repositories/Summit/DoctrineSpeakersSummitRegistrationPromoCodeRepository.php

```diff
@@ -58,7 +58,9 @@ protected function getOrderMappings()
             'email'     => 'm.email',
             'full_name' => <<<SQL
 LOWER(CONCAT(m.first_name, ' ', m.last_name))
-SQL
+SQL,
+            'email_sent' => 'o.sent',
+            'redeemed'   => 'o.redeemed'
```

Comment:
this is not correct 
check https://github.com/OpenStackweb/summit-api/blob/094b0cf35e28ab6474336151e24fe6588ec5e473/app/Models/Foundation/Summit/Registration/PromoCodes/Traits/SpeakersPromoCodeTrait.php#L306 is a calculated prop for speakers promo code